### DISCUSSION TITLE
Use shared redis across classes/modules

### DISCUSSION THREAD

**Discussion by andreasdc** [bcit-ci/CodeIgniter#6207] [Q&A]
I want to bring load times to the top, 
`    public function __construct() {
        $this->redis = new \CI_Predis\Redis();
    }
`
How can I use redis across multiple classes/modules? Because loading models and cache driver take some time and I think it can be optimized.

**Comment by MeydhiXploit** [comment]
what is __construct()? i dont understand the first

**Reply by andreasdc** [reply]
I have this function in models, it loads redis.
